dayrepo/snippets/views.py

```diff
@@ -113,7 +113,7 @@ def post(self, request):
         )
         snippet.save()
 
-        # DutiesTrouble form
+        # DutiesTrouble form　業務トラブルフォーム保存
```

Comment:
```suggestion
        # 業務トラブルフォーム保存
```
英語と日本語で意味を重複させる意味はない